boards/arm/96b_carbon/96b_carbon_defconfig

```diff
@@ -41,10 +41,3 @@ CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=7
 CONFIG_CLOCK_STM32_AHB_PRESCALER=1
 CONFIG_CLOCK_STM32_APB1_PRESCALER=2
 CONFIG_CLOCK_STM32_APB2_PRESCALER=1
-
-# BT SPI configuration when BT is selected
-CONFIG_BT_SPI=y
```

Comment:
We should leave this.

Reply:
Why should we leave it? BT_SPI will be enabled in board Kconfig.defconfig it BT is enabled

Reply:
BT_SPI is not enabled in board Kconfig.defconfig right now.